Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -313,17 +359,389 @@ def zoom_list_user_messages(self, user_id: str, date_arg: datetime, from_arg: da
+    demisto.debug(f"request: {request}")
+    # time.sleep(10)
+    # demisto.debug(f"after sleep: {request}")
+    event_type = request['event']
+    payload = request['payload']
+    try:
+        if event_type == 'endpoint.url_validation' and SECRET_TOKEN:
+            demisto.debug(f"validate: {payload}")
+            res = await event_url_validation(payload)
+            return JSONResponse(content=res)
+
+        if event_type == 'interactive_message_actions':
+            if payload.get('actionItem', False):
+                action = payload['actionItem']['value']
+            elif payload.get('selectedItems', False):
+                action = payload['selectedItems'][0]['value']
+            else:
+                return Response(status_code=status.HTTP_400_BAD_REQUEST)
+            message_id = payload['messageId']
+            account_id = payload['accountId']
+            robot_jid = payload['robotJid']
+            to_jid = payload['toJid']
+            user_name = payload['userName']
+            demisto.debug(f"message_id: {message_id}")
+            entitlement_reply = await check_and_handle_entitlement(action, message_id, user_name)  # type: ignore
+            if entitlement_reply:
+                await process_entitlement_reply(entitlement_reply, account_id, robot_jid, to_jid, user_name, action)
+                demisto.updateModuleHealth("")
+            demisto.debug(f"button was clicked {action} on message {message_id}")
+            return Response(status_code=status.HTTP_200_OK)
+        if event_type == "chat_message.sent" and MIRRORING_ENABLED:
+            await handle_mirroring(payload)
+            return Response(status_code=status.HTTP_200_OK)
+        else:
+            return Response(status_code=status.HTTP_400_BAD_REQUEST)
+    except Exception as e:
+        await handle_listen_error(f'Error occurred while handel response from Zoom: {e}')
+
+
 def test_module(client: Client):
     """Tests connectivity with the client.
     Takes as an argument all client arguments to create a new client
     """
     try:
-        # running an arbitrary command to test the connection
-        client.zoom_list_users(page_size=1, url_suffix="users")
+        if MIRRORING_ENABLED and (not LONG_RUNNING or not SECRET_TOKEN or not client.bot_client_id or not
+                                  client.bot_client_secret or not client.botJid):
+            demisto.error('Mirroring is enabled, however long running is disabled or bot secretToken not exist.',
+                          'For mirrors to work correctly, long running must be enabled and you must provide the ',
+                          'secret token, Bot JID, bot client and secret id')
+
+        if client.access_token:
+            client.zoom_list_users(page_size=1, url_suffix="users")
+        if client.bot_access_token:
+            json_data = {
+                "robot_jid": client.botJid,
+                "to_jid": "blabla@conference.xmpp.zoom.us",
```

Comment:
Is this a standard value for testing? If not, use `foo`.
```suggestion
                "to_jid": "foo@conference.xmpp.zoom.us",
```